ClassicPress-research/classic-commerce
# Change WC to CC on advanced settings page

### All Submissions:


### Changes proposed in this Pull Request:

Change one occurrence of WC to CC on advanced settings page, located in file: class-wc-settings-advanced.php 

### How to test the changes in this Pull Request:

1. Check out my PR for changes to file class-wc-settings-advanced.php
2. Load onto a test site and visit the advanced settings section.

### Other information:


### Changelog entry

Change WC to CC on advanced settings page.
